_aicspylibczi/TypedImage.h

```diff
@@ -26,7 +27,7 @@ namespace pylibczi {
        * @param m_index_ The mosaic index for the image, this is only relevant if the file is a mosaic file.
        */
       TypedImage(std::vector<size_t> shape_, libCZI::PixelType pixel_type_, const libCZI::CDimCoordinate* plane_coordantes_,
-          libCZI::IntRect box_, T* mem_ptr_, int m_index_)
+          libCZI::IntRect box_, T *mem_ptr_, int m_index_)
```

Comment:
something to check: consistent formatting of pointer asterisks, do you want them close to the type or close to the variable name?  The line above is doing the opposite of this line.